ibm_catalog.json

```diff
@@ -172,74 +164,269 @@
               ]
             },
             {
-              "key": "iam_engine_enabled"
+              "key": "skip_iam_authorization_policy"
             },
             {
-              "key": "iam_engine_name"
+              "key": "allowed_network",
+              "options": [
+                {
+                  "displayname": "private-only",
+                  "value": "private-only"
+                },
+                {
+                  "displayname": "public-and-private",
+                  "value": "public-and-private"
+                }
+              ]
             },
             {
-              "key": "public_cert_engine_enabled"
+              "key": "existing_resource_group_name"
```

Comment:
This should be a resource group picker. See https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/5a7ec37b7508dd05d8f9d86e6ef0ec301b058381/ibm_catalog.json#L81-L89